website/data/docs-nav-data.json

```diff
@@ -403,16 +403,42 @@
         "path": "api-gateway/consul-api-gateway-install"
       },
       {
-        "title": "Technical Specifications",
-        "path": "api-gateway/tech-specs"
```

Comment:
Looks like this would need a redirect added from `api-gateway/tech-specs` to `api-gateway/consul-api-gateway-install` (which FWIW feels like a kinda gross, redundant path)

Reply:
We could change `api-gateway/consul-api-gateway-install` to `api-gateway/install` while we're in here. I agree it feels gross right now. That install page is the oldest page in these docs IIRC, and none of our other pages include that duplicate prefix

Reply:
Not opposed to changing it, but I was just following the menu outlined here https://docs.google.com/document/d/12BQVxOGMUE73FicXagW02YujB29Hm2x2T7WRmSeShHo/edit#. Not sure how strictly we need to adhere to whats outline in that document.

Reply:
Changing back to having technical specs being a separate page after a note from Jeff